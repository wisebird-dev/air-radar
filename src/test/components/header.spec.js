import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Header from '../../components/header'

test('Header renders correctly', async () => {
  const tree = await renderer.create(<Header />).toJSON()

  expect(tree).toMatchSnapshot()
})