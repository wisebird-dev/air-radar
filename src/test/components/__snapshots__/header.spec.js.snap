// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
.c0 {
  font-size: 25px;
}

<h1
  className="c0"
/>
`;
